features/matchers/developer_uses_trigger_matcher.feature

```diff
@@ -0,0 +1,163 @@
+Feature: Developer uses trigger matcher
+  As a Developer
+  I want a trigger matcher
+  In order to validate triggered exceptions against my expectations
+
+  Scenario: "Trigger" alias matches using the trigger matcher with explicit method name
+    Given the spec file "spec/Matchers/TriggerExample1/FooSpec.php" contains:
+    """
+    <?php
+
+    namespace spec\Matchers\TriggerExample1;
+
+    use PhpSpec\ObjectBehavior;
+    use Prophecy\Argument;
+
+    class FooSpec extends ObjectBehavior
+    {
+        function it_triggers_an_error_because_it_is_deprecated()
+        {
+            $this->shouldTrigger(E_USER_DEPRECATED)->during('doDeprecatedStuff');
+        }
+    }
+    """
+
+    And the class file "src/Matchers/TriggerExample1/Foo.php" contains:
+    """
+    <?php
+
+    namespace Matchers\TriggerExample1;
+
+    class Foo
+    {
+        public function doDeprecatedStuff()
+        {
+            trigger_error('Foo', E_USER_DEPRECATED);
+        }
+    }
+    """
+
+    When I run phpspec
+    Then the suite should pass
+
+  Scenario: "Trigger" alias matches using the trigger matcher with implicit method name
```

Comment:
I don't think this scenario's necessary - you're testing that the `during*` magic works, which is the responsibility of `DuringCall` (which I don't think you've modified).

You can just use one syntax in the scenarios.